pkg/logs/diagnostic/format.go

```diff
@@ -32,18 +32,11 @@ func (l *logFormatter) Format(m *message.Message, eventType string, redactedMsg
 		hname = "unknown"
 	}
 
-	ts := time.Now().UTC()
-	// TODO(remy): should we consider renaming the "Timestamp: %s" to mention
-	// it's only concerning the serverless agent?
-	if !m.ServerlessExtra.Timestamp.IsZero() {
-		ts = m.ServerlessExtra.Timestamp
-	}
-
 	return fmt.Sprintf("Integration Name: %s | Type: %s | Status: %s | Timestamp: %s | Hostname: %s | Service: %s | Source: %s | Tags: %s | Message: %s\n",
 		m.Origin.LogSource.Name,
 		m.Origin.LogSource.Config.Type,
 		m.GetStatus(),
-		ts,
+		time.Now().UTC(),
```

Comment:
Will this change the timestamp on serverless log messages? Help me understand this update.

Reply:
Okay, I think I understand now. Serverless stream logs will no longer be processed using this method, so you are removing the serverless specific code.

Reply:
@purple4reina Yes, the serverless-agent will depend on `streamlogs.Formatter`. We no longer need to consider `diagnostic.logFormatter` being called by the serverless-agent.